### PR TITLE
Update security considerations

### PR DESCRIPTION
**Pull request by sustefil**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,6 +93,25 @@ We follow the official OASIS specifications in order to provide as much acceptan
 
 ### Security Considerations
 
+These are the TOP OWASP categories of vulnerabilities which potentially affect the CVRF-CSAF-Converter.
+We are omitting those which do not apply (most of them), since it's a plain command-line tool (e.g. authentication failures)
+
+#### [A03:2021 – Injection](https://owasp.org/Top10/A03_2021-Injection/)
+The XML input for the converter is strictly validated for CVRF. The converter rejects invalid inputs.
```
Comment (tschmidtb51) [inline README.md:100]: CVRF is officially called CSAF CVRF 1.2... 
```suggestion
The XML input for the converter is strictly validated for [CSAF CVRF 1.2](https://docs.oasis-open.org/csaf/csaf-cvrf/v1.2/cs01/csaf-cvrf-v1.2-cs01.html). The converter rejects invalid inputs.

File: README.md [hunk 2]
```diff
@@ -93,6 +93,25 @@ We follow the official OASIS specifications in order to provide as much acceptan
...
+
+However, there is a known issue for inserting HTML with code/script, which could be executed by a CSAF consumer: 
+[Encode HTML in JSON output](https://github.com/csaf-tools/CVRF-CSAF-Converter/issues/5)
+
+#### [A06:2021 - Vulnerable and Outdated Components](https://owasp.org/Top10/A06_2021-Vulnerable_and_Outdated_Components/)
+An automated SAST scanning tool like Github Dependabot can be used to scan the libraries for vulnerabilities.
```
Comment (tschmidtb51) [inline README.md:106]: Maybe, we should set up CodeQL as a GitHub action to ensure that?

**Review by tschmidtb51 — COMMENTED** [review]
One minor issue and a suggestion to setup CodeQL and link it in the README.

File: README.md (hunk 2 above)
Comment (sustefil) [reply README.md:106]: Sure, set in #91 and passed, have a look.

Then I would rewrite:
"An automated SAST scanning tool like Github Dependabot can be used to scan the libraries for vulnerabilities."
to
"A CodeQL action is set in this project to spot vulnerabilities in 3rd party libraries."

Comment (tschmidtb51) [reply README.md:106]: Agreed. Please merge #92 before to be able to refer to it with a link.

**Pull request merged**